README.md

```diff
@@ -31,7 +31,7 @@
 ## Installation
 
 ```bash
-npm i -D nestjs-pact @pact-foundation/pact
+npm i -D nestjs-pact @pact-foundation/pact@^9.0.0
```

Comment:
will that pick up 9.0.0 on install, or the latest minor version which is 9.18.1

Reply:
```
❯ npm i -D nestjs-pact @pact-foundation/pact@^9.0.0
zsh: no matches found: @pact-foundation/pact@^9.0.0
```

this works (the ^ isn't recognised ) and pointing a user to the older version will mean it won't support m1 platforms.

```
❯ npm i -D nestjs-pact @pact-foundation/pact@9.18.1
```